sections/designRules.md

```diff
@@ -179,13 +179,15 @@ A resource that corresponds to a single conceptual entity is referred to as a [=
             <pre class="nohighlight example-correct">https://api.example.org/v1/gebouwen?typeGebouw=woning</pre>
             <p>URI query key not using camelCase (incorrect):</p>
             <pre class="nohighlight example-incorrect">https://api.example.org/v1/gebouwen?type-gebouw=woning</pre>
+            <p>URI query key starts with digit (incorrect):</p>
+            <pre class="nohighlight example-incorrect">https://api.example.org/v1/gebouwen?2ndReviewer=alice</pre>
          </div>
       </dd>
       <dt>How to test</dt>
       <dd>
          Loop all resource paths in the OpenAPI Description and check that all query keys use letters, digits in camelCase. You can use the following regex for each query key:
          <div class="example">
-            <pre><code>^[a-z0-9]+[a-zA-Z0-9]*$</code></pre>
+            <pre><code>^\$?[a-z][a-z0-9]*([A-Z][\w0-9]*)*$</code></pre>
```

Comment:
```suggestion
            <pre><code>^\$?[a-z][a-z\d]*([A-Z][a-z\d]*)*$</code></pre>
```
Is `\w` breder dan bedoeld?

Reply:
Het idee was dat meerdere hoofdletters achter elkaar ook oke zouden zijn (is ambigu bij camelCase). Maar dat kan met jouw suggestie ook, dus laten we daar ook gewoon voor gaan.